src/core/Alert/Alert/Alert.test.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { Alert } from './Alert';
+import { axeTest } from '../../../utils/test';
+
+describe('children', () => {
+  const alertWithElementChild = (
+    <Alert closeText="Close">
+      <div>Test alert</div>
+    </Alert>
+  );
+
+  it('has the given content', () => {
+    const { container } = render(alertWithElementChild);
+    expect(container.firstChild).toHaveTextContent('Test alert');
+  });
+
+  it('should match snapshot', () => {
+    const { container } = render(alertWithElementChild);
+    expect(container).toMatchSnapshot();
+  });
+});
+
+describe('classnames', () => {
+  const customClassAlert = (
+    <Alert closeText="Close" className="custom-class">
+      Test content
+    </Alert>
+  );
+
+  it('contains base classname', () => {
+    const { container } = render(customClassAlert);
+    expect(container.firstChild).toHaveClass('fi-alert');
+  });
+
+  it('contains custom classname', () => {
+    const { container } = render(customClassAlert);
+    expect(container.firstChild).toHaveClass('custom-class');
+  });
+});
```

Comment:
This whole `describe` block could be moved inside the `props` describe block. 
And to be changed to singular as the prop name is `className` instead of `classnames`.